### PR TITLE
feat: use `@jsse/pbfont` instead of `@mapbox/glyph-pbf-composite`

### PR DESCRIPTION
**Pull request by prashis**
- Use `@jsse/pbfont` instead of `@mapbox/glyph-pbf-composite`
- Fixes https://github.com/maptiler/tileserver-gl/issues/1268

### CODE REVIEW

File: src/utils.js [hunk 1]
```diff
@@ -4,7 +4,7 @@ import path from 'path';
 import fsPromises from 'fs/promises';
 import fs, { existsSync } from 'node:fs';
 import clone from 'clone';
-import glyphCompose from '@mapbox/glyph-pbf-composite';
+import * as glyphCompose from '@jsse/pbfont';
```
Comment (acalcutt) [inline src/utils.js:7]: Can we just import the named combine function 'combine'

Comment (prashis) [reply src/utils.js]: Yes, Have made the changes accordingly. :)

**Comment by jessekrubin** [comment]
How performant is this tile server? just curious.

**Comment by acalcutt** [comment]
I'd say performance it a tough thing to judge, and tileserver-gl has a lot of different configuration options that change performance. you can see an example of my server at https://tiles.wifidb.net , which is just npm running on a Ubuntu 22.04  virtual server with HA proxy in between, and a fallback server on a rasperry pie. the later pi is a bit slow rendering, but good for a backup/

Just a note on this PR. I am having a bit of issues with my permission on the repo, so it might take some time for me to get it fixed with maptiler and get this approved.

**Review by acalcutt — APPROVED** [review]

**Pull request merged**